packages/puppeteer-core/src/common/bidi/Frame.ts

```diff
@@ -64,17 +64,18 @@ export class BidiFrame extends Frame {
     this._id = this.#context.id;
     this._parentId = parentId ?? undefined;
 
-    const puppeteerRealm = context.createSandboxRealm(UTILITY_WORLD_NAME);
     this.sandboxes = {
-      [MAIN_SANDBOX]: new Sandbox(context, timeoutSettings),
-      [PUPPETEER_SANDBOX]: new Sandbox(puppeteerRealm, timeoutSettings),
+      [MAIN_SANDBOX]: new Sandbox('', this, context, timeoutSettings),
```

Comment:
This is incorrect is should be null. A sandbox with name empty may be different than the main one.